test/E2ETests/Automation.CLI/CLITestBase.e2e.cs

```diff
@@ -17,7 +17,8 @@ public partial class CLITestBase : TestContainer
 {
     private static VsTestConsoleWrapper s_vsTestConsoleWrapper;
     private DiscoveryEventsHandler _discoveryEventsHandler;
-    private RunEventsHandler _runEventsHandler;
+
+    protected RunEventsHandler RunEventsHandler { get; set; }
```

Comment:
Not sure why you changed this to a property (but I am fine with it). Is it possible to make the setter private?